src/test/java/com/shinesolutions/aemorchestrator/service/AemInstanceHelperServiceTest.java

```diff
@@ -182,8 +180,6 @@ public void testGetPublishIdToSnapshotFrom() throws Exception {
         when(awsHelperService.getLaunchTime(instanceId)).thenReturn(originalDate);
         when(awsHelperService.getLaunchTime("extra-89351")).thenReturn(originalPlusOneDate);
 
-        when(httpUtil.isHttpGetResponseOk(anyString())).thenReturn(true);
```

Comment:
Unnecessary stubbing `httpUtil.isHttpGetResponseOk`, is not invoked when `aemHelperService.getAemComponentInitState` is called.